apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -635,7 +635,7 @@ def create_table_for_html_display(
             + str(m)
             + '"></td>'
         )
-        if score_message is not None:
```

Comment:
score_message could be an empty string, I assumed that was the case this is supposed to hit.

Reply:
Would `if score_message:` be more Pythonic?

Reply:
Apparently it is: https://stackoverflow.com/a/9573283. TIL!

Reply:
Resolved in 87d8d3e1e5023ebc1a4a6af1edda3dfccfdc41ff.